zap/src/main/java/org/parosproxy/paros/db/RecordAlertTag.java

```diff
@@ -0,0 +1,48 @@
+package org.parosproxy.paros.db;
```

Comment:
Just noticed that spotlessApply did not add a license to the classes in this package - should one be added manually?

Reply:
Ya it’s probably a package that is excluded because of being Paros code, just add the standard ZAP header.

Reply:
Better add new classes in the ZAP packages.